pkg/services/notification/resource_control/notification.go

```diff
@@ -140,6 +150,7 @@ func SplitNotificationIntoTasks(ctx context.Context, notification *models.Notifi
 			task := models.NewTask(
 				notification.NotificationId,
 				jsonutil.ToString(directive),
+				address.NotifyType,
```

Comment:
notifyType already exists in directive, is it needed here?

Reply:
yes, need use it to distinguish different task if we add other type such as sms,wechat in the future.